STUDIO/org.openl.rules.test/test/org/openl/syntax/LiteralCompilationTest.java

```diff
@@ -0,0 +1,26 @@
+package org.openl.syntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import org.openl.CompiledOpenClass;
+import org.openl.rules.runtime.RulesEngineFactory;
+
+class LiteralCompilationTest {
+
+    @Test
+    void testLiteralError() {
+        RulesEngineFactory<?> engineFactory = new RulesEngineFactory<>(
+                "./test-resources/IncorrectVocabulary/EPBDS-14557.xlsx");
+        engineFactory.setExecutionMode(true);
+        CompiledOpenClass compiledOpenClass = engineFactory.getCompiledOpenClass();
+
+        assertTrue(compiledOpenClass.hasErrors(), "Expected an error in the project");
+        assertEquals(1, compiledOpenClass.getAllMessages().size(), "Must have only one error");
+        assertEquals("Object 'F' is outside of valid domain 'Gender'. Valid values: [Male, Female]",
+                compiledOpenClass.getAllMessages().iterator().next().getSummary(),
+                "Incorrect error message");
+    }
+}
```

Comment:
Remove this Java test and create the files: 
`EPBDS-14557_Vocabulary_validation.xlsx` - your excel file
`EPBDS-14557_Vocabulary_validation.xlsx.msg.txt` - expected error messages

in https://github.com/openl-tablets/openl-tablets/blob/master/STUDIO/org.openl.rules.test/test-resources/functionality/

Add the case when there several different vocabularies is used in the single method. Like
`calculate("F", 125)` for `calculate(Gender, Level)`
where `Level` is also vocabulary of [100, 200, 300] numbers.

Reply:
do the same checks for the signature `calculate2(String, Gender[], Level[])`